src/components/tasks/card/index.tsx

```diff
@@ -85,7 +85,7 @@ const Card: FC<Props> = ({
   
   function handleProgressColor(percentCompleted: number, startedOn: string, endsOn: string): string {
     const percentageofDays = getPercentageOfDays(startedOn, endsOn)
-    if(percentageofDays >= percentCompleted) {
+    if(percentCompleted === 100 && (percentageofDays >= percentCompleted || percentageofDays < 0)) {
```

Comment:
when you refactor the code, please make sure that this condition also changes `percentageofDays >= percentCompleted || percentageofDays < 0` it will return true even if  `percentageofDays >= percentCompleted` returns false

Reply:
updated the coditions. Do they make more sense now?